src/main/resources/application.properties

```diff
@@ -2,7 +2,6 @@
 spring.datasource.url=jdbc:mysql://localhost/customer_website?useSSL=false&allowPublicKeyRetrieval=true&serverTimezone=UTC
 spring.datasource.username=root
 spring.datasource.password=${PASSWORD}
-
```

Comment:
An empty line could be used as a "logical separator" like a paragraph in a book. 
I use them in properties and code and pox.xml/build.gradle  to groul something related and separate from next block. I believe it improves readability